test/automation/src/sql/notebook.ts

```diff
@@ -115,6 +120,18 @@ export class Notebook {
 		return (await this.code.waitForElements('div.notebook-cell', false)).map(cell => cell.attributes['id']);
 	}
 
+	// Code Cell Actions
+
+	async waitForSuggestionWidget(): Promise<void> {
+		const suggestionWidgetSelector = 'div.editor-widget.suggest-widget';
+		await this.code.waitForElement(suggestionWidgetSelector);
+	}
+
+	async waitForSuggestionResult(expectedResult: string): Promise<void> {
+		const expectedResultSelector = `div.monaco-list-row.focused[aria-label="${expectedResult}"]`;
```

Comment:
Can this be scoped down a bit further? It's not likely to be a problem, but these classes are pretty generic so it would be nice if we could scope it down to specifically only searching under the suggestion widget.